src/Mobs/Monster.cpp

```diff
@@ -435,19 +428,41 @@ void cMonster::CalcLeashActions()
 			SetLeashToPos(nullptr);
 		}
 	}
-	else if (IsLeashed())  // Mob is already leashed to an entity: follow it.
+
+	if (!IsLeashed())
+	{
+		return;
+	}
+
+	static const double LeashNaturalLength = 5.0;   // Length at equilibrium
```

Comment:
The comment here suggests that when entities are closer than the NaturalLength, they are pushed away. Perhaps "Length where the leash begins to pull" would be better? That doesn't sound great either so idk.

Reply:
Yes, equilibrium isn't quite right here.  I'll change it.